src/github/middleware.ts

```diff
@@ -235,12 +237,20 @@ export default (
 			try {
 				await callback(context, jiraClient, util);
 			} catch (err) {
-				context.log.error(
-					err,
-					`Error processing the event for Jira hostname '${jiraHost}'`
-				);
-				emitWebhookFailedMetrics(webhookEvent);
-				context.sentry?.captureException(err);
+				const isWarning = warnOnErrorCodes.find(code => err.message.includes(code));
+				if(!isWarning) {
+					context.log.error(
+						err,
+						`Error processing the event for Jira hostname '${jiraHost}'`
+					);
+					emitWebhookFailedMetrics(webhookEvent);
+					context.sentry?.captureException(err);
+				} else {
+					context.log.info(
+						err,
+						`Error processing the event for Jira hostname '${jiraHost}'`
```

Comment:
If we are breaking this out into an error and a warning should we change the messaging here to make that a little clearer? Something like: `Warning: failed to process event for the Jira hostname '${jiraHost}'`. This will just make it easier to distinguish when looking through the logs for `github.webhooks`

- [ ] update message for `github.webhooks` warning